pkg/alpha/precheck/task.go

```diff
@@ -135,3 +136,53 @@ func (g *GetAllNodesK8sVersion) Execute(runtime connector.Runtime) error {
 	host.GetCache().Set(common.NodeK8sVersion, nodeK8sVersion)
 	return nil
 }
+
+type GetMasterK8sVersion struct {
+	common.KubeAction
+}
+
+func (g *GetMasterK8sVersion) Execute(runtime connector.Runtime) error {
+	apiserverVersion, err := runtime.GetRunner().SudoCmd(
+		"cat /etc/kubernetes/manifests/kube-apiserver.yaml | grep 'image:' | rev | cut -d ':' -f1 | rev",
+		false)
+	if err != nil {
+		return errors.Wrap(err, "get current kube-apiserver version failed")
+	}
+	g.PipelineCache.Set(common.K8sVersion, apiserverVersion)
+	g.KubeConf.Cluster.Kubernetes.Version = apiserverVersion
```

Comment:
Could you please tell me why need to assign a new value to this `g.KubeConf.Cluster.Kubernetes.Version`?